api/v1alpha1/openstacknodeimagerelease_types.go

```diff
@@ -40,42 +40,12 @@ type OpenStackNodeImageReleaseSpec struct {
 type OpenStackNodeImage struct {
 	URL string `json:"url" yaml:"url"`
 	// CreateOpts represents options used to create an image.
-	CreateOpts *CreateOpts `json:"createOpts" yaml:"createOpts"`
+	CreateOpts *images.CreateOpts `json:"createOpts" yaml:"createOpts"`
 }
 
 // CreateOpts represents options used to create an image.
 // CreateOpts attributes are taken from github.com/gophercloud/gophercloud/openstack/imageservice/v2/images.
 // Consider.
-type CreateOpts struct {
```

Comment:
@NotTheEvilOne 
this should be copied for now, as the build failed if you import CreateOpts 
```
# [redacted secret cspo-cluster-stack-variables:git-provider].com/sovereignCloudStack/cluster-stack-provider-openstack/api/v1alpha1
api/v1alpha1/zz_generated.deepcopy.go:243:9: (*in).DeepCopyInto undefined (type *images.CreateOpts has no field or method DeepCopyInto)
```

Reply:
revert 64d1bde899457a067fed90cedc2567ff687673d5